less/home-page.less

```diff
@@ -46,9 +68,20 @@
     display: flex;
     justify-content: center;
 
+    @media @phone {
+      .center-align;
+      flex-direction: column-reverse;
```

Comment:
Nice. This was a good way to do it. Another way would've been to make a bigger div for these elements, and make that a flex container and then use `order: (number)` for the children elements to move them around in there.